lib/deprecation-cop-view.coffee

```diff
@@ -36,6 +36,13 @@ class DeprecationCopView extends ScrollView
     @subscriptions.add atom.packages.onDidActivatePackage (pack) =>
       @refreshSelectorsButton.show() if pack.isTheme()
 
+    @subscriptions.add atom.keymaps.onDidReloadKeymap (event) =>
+      @refreshSelectorsButton.show() if event.path is atom.keymaps.getUserKeymapPath()
+
+    try
+      userStylesheetFile = new File(atom.styles.getUserStyleSheetPath())
+      @subscriptions.add userStylesheetFile.onDidChange => @refreshSelectorsButton.show()
```

Comment:
This could use `atom.styles` listeners the same as `deprecation-cop-status-bar-view.coffee` right?